src/components/Display.js

```diff
@@ -0,0 +1,106 @@
+    dispatch(removeBookSuccess(e.target));
+  };
+
+  const myBooks = useSelector((state) => state.books.books);
+
+  const AddBookCard = () => (
+    <div>
+      {myBooks.map((book) => {
+        const randomNum = Math.floor(Math.random() * 100);
+        const barstyle = {
+          width: `${randomNum}%`,
+        };
+
+        return (
+          <div className="card" key={book.id}>
+            <div className="card-body">
+              <div className="container">
+                <div className="row">
+                  <div className="col">
+                    <p className="fs-6 text-secondary">{book.category}</p>
+                    <p className="fs-5 fw-bold">{book.title}</p>
+                    <p className="fs-5 text-primary">{book.author}</p>
+                    <div className="container p-0">
+                      <div className="row">
+                        <div className="col">
+                          Comments
+                        </div>
+                        <div className="col">
+                          <button type="button" onClick={removeBookBtn} id={book.id} className="buttonRemove">Remove</button>
+                        </div>
+                        <div className="col">
+                          Edit
+                        </div>
+                      </div>
+                    </div>
+                  </div>
+                  <div className="col">
+                    {randomNum}
+                    % Completed
+                    <div className="progress">
+                      <div className="progress-bar progress-bar-striped progress-bar-animated" role="progressbar" aria-label="bar" aria-valuenow="75" aria-valuemin="0" aria-valuemax="100" style={barstyle} />
+                    </div>
+                  </div>
+                  <div className="col">
+                    <p>Current chapter</p>
+                    <p>
+                      Chapter&nbsp;
+
+                      {Math.floor(Math.random() * 100)}
+                    </p>
+                    <button type="button" className="btn btn-primary">UPDATE PROGRESS</button>
+                  </div>
+                </div>
+              </div>
+            </div>
+          </div>
+        );
+      })}
+    </div>
+  );
+
+  return (
+    <div className="container align-middle pt-5">
+      <div className="container my-5 align-middle" id="formContainer">
+        <div className="card">
+          <div className="card-header">
+            <div className="container">
+              <div className="row">
+                <div className="col">
+                  Bookstore CMS
+                </div>
+                <div className="col">
+                  <Link className="links" to="/">BOOKS</Link>
+                </div>
+                <div className="col">
+                  <Link className="links" to="/categories">CATEGORIES</Link>
+                </div>
+              </div>
+            </div>
+          </div>
```

Comment:
**Changes Required**
- [ ] The navbar should look like the Zeplin mock up. Let us try yo make it as close as possible. This will make us better front-end developers.

**Your Design**
![your-navbar](https://user-images.githubusercontent.com/7220167/132073416-60a03249-254f-4dd4-9631-4c893981b289.png)

**Expected Design**
![zeplin-navbar](https://user-images.githubusercontent.com/7220167/132073429-3f09b1ff-21a8-4ac9-a099-46a32729290c.png)